style.css

```diff
@@ -0,0 +1,89 @@
+li {
+  display: flex;
+  height: 28px;
+  width: 90px;
+  left: 12px;
+  top: 10px;
+  border-radius: nullpx;
+  font-family: Inter, sans-serif;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.portfolio {
+  padding-left: 0;
+}
+
+.about {
+  margin-left: 5%;
+}
+
+.contact {
+  padding-right: 0;
+  margin-left: 10%;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+li a:hover {
+  background-color: rgb(109, 171, 207);
+}
+
+p {
+  display: flex;
+  height: 164px;
+  width: 327px;
+  left: 24px;
+  top: 314px;
+  margin-top: 58px;
+  border-radius: nullpx;
+  font-family: Inter, sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.toolbar {
+  margin-left: 280px;
+  height: 12px;
+  width: 18px;
+  margin-top: -14px;
+  border-radius: 0;
+}
+
+h1 {
+  display: flex;
+  height: 70px;
+  width: 327px;
```

Comment:
- [ ] To make your project more responsive refrain from setting the width to less than 100%. 100%  means the targeted container is fully responsive. I would encourage you to remove all parts of the CSS code that has a width of less than 100% in your code or set the width to 100% everywhere you have put a width property in your CSS for a more responsive site.